File: JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands2/lobby/training/IdentifyResponse.java
package bluffinmuffin.protocol.commands2.lobby.training;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands2.lobby.AbstractLobbyResponse;

public class IdentifyResponse extends AbstractLobbyResponse<IdentifyCommand>
{
    @Override
    protected String getCommandName()
    {
        return IdentifyResponse.COMMAND_NAME;
    }
    
    public static String COMMAND_NAME = "lobbyIDENTIFY_TRAINING_RESPONSE";
    
    private final boolean m_isOK;
    
    public IdentifyResponse(StringTokenizer argsToken)
    {
        super(new IdentifyCommand(argsToken));
        m_isOK = Boolean.parseBoolean(argsToken.nextToken());
    }
    
    public IdentifyResponse(IdentifyCommand command, boolean ok)
    {
        super(command);
        m_isOK = ok;
    }
    
    @Override
    public void encode(StringBuilder sb)
    {
        super.encode(sb);
        append(sb, m_isOK);
    }
    
    public boolean isOK()
    {
        return m_isOK;
    }
}

File: C#/PokerProtocol/Commands/Lobby/Training/IdentifyResponse.cs
using System;
using System.Collections.Generic;
using System.Text;
using EricUtility;
using EricUtility.Networking.Commands;
using Newtonsoft.Json.Linq;

namespace PokerProtocol.Commands.Lobby.Training
{
    public class IdentifyResponse : AbstractLobbyResponse<IdentifyCommand>
    {
        public static string COMMAND_NAME = "lobbyIDENTIFY_TRAINING_RESPONSE";

        public bool OK { get; set; }

        public IdentifyResponse()
            : base()
        {
        }

        public IdentifyResponse(IdentifyCommand command, bool ok)
            : base(command)
        {
            OK = ok;
        }
    }
}
